server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -245,6 +246,21 @@ pub struct ChangelogRepository<'a> {
 
 type ChangelogJoin = (ChangelogRow, Option<NameLinkRow>);
 
+impl ChangelogRow {
+    pub fn from_join((row, name_link): (ChangelogRow, Option<NameLinkRow>)) -> Self {
+        ChangelogRow {
```

Comment:
I really like these helpers, in other repositories we call them `to_domain`, with some variations

Reply:
Yeah, I wasn't quite sure here, I thought `to_domain` was more for mapping to the non `-Row` structs, but could call this the same I suppose